### PR TITLE
Update phone with clarify Contact and expand UserInfo schemas

### PR DESCRIPTION
**Pull request by SERDUN**

### CODE REVIEW

File: packages/webtrit_api/test/webtrit_api_test.dart [hunk 1]
```diff
@@ -526,9 +526,12 @@ void main() {
         return Response(
           jsonEncode({
             'sip': {
-              'display_name': 'display_name_1',
```
Comment (ycherniavskyi) [inline packages/webtrit_api/test/webtrit_api_test.dart:529]: This field didn't remove.

File: packages/webtrit_api/test/webtrit_api_test.dart [hunk 2]
```diff
@@ -540,9 +543,9 @@ void main() {
               'ext': '0001',
               'additional': ['380441234567', '34911234567'],
             },
+            'email': 'email_1',
             'first_name': 'first_name_1',
             'last_name': 'last_name_1',
```
Comment (ycherniavskyi) [inline packages/webtrit_api/test/webtrit_api_test.dart:548]: Add `alies_name` field here.

**Review by ycherniavskyi — CHANGES_REQUESTED** [review]
Within this PR also must be added DB migration for renaming `displayName` to `aliasName`.

**Review by ycherniavskyi — CHANGES_REQUESTED** [review]
Also, it seems that you forgot to update the test data with values for added and renamed fields.

**Comment by SERDUN** [comment]
It seems better not to use idea refactoring when replacing the name and do it by hand, for correctly style

**Review by ycherniavskyi — APPROVED** [review]

**Pull request merged**